frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -66,6 +66,21 @@ <h3 class="flex justify-center sm:hidden">Create new project</h3>
                 </mat-radio-group>
               </div>
             </fieldset>
+            <fieldset>
+              <div class="flex flex-col">
+                <legend>Project type</legend>
+                <mat-radio-group formControlName="type" class="flex flex-col">
+                  <mat-radio-button
+                    *ngFor="
```

Comment:
Please use the new control flow: https://angular.dev/guide/templates/control-flow#repeat-content-with-the-for-block